Create ISSUE_TEMPLATE.md

Aerijo opened the pull request:
This will provide a template for all new issue submissions, guiding people on how to make new submissions as informative as possible in the first post. No more asking how to reproduce the issue: they will know to have an example ready for anyone to copy and paste to immediately work with.

I put in the first line to feel more welcoming, but I'm open to any suggestions or changes (to any of it).

yudai-nkt commented on issue_template.md:
```diff
@@ -0,0 +1,29 @@
+<!--
+Thanks for notifying us of a new issue!
+
+Before you submit, please make sure you do the following:
```
I think these instructions can be a TODO list using a [task list](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments) rather than a comment (comment cannot be seen in preview section). Good example I know would be [a template for PRs in sindresorhus/atom-editorconfig](https://github.com/sindresorhus/atom-editorconfig/blob/v2.2.2/.github/pull_request_template.md).

yudai-nkt replied:
"please make sure you have done the following" sounds more natural to me, but I'm not sure if it really is (I'm not a native English speaker :-)).

Aerijo commented:
Changed the tense, added the prerequisites section, and renamed file. Only using one checkbox to make it easier to use. (I get annoyed just having to delete a space and then put an X in one box, let alone several).

yudai-nkt commented:
You actually don't need to replace a whitespace with an `x`, but can check each item using a mouse like this:

![mouse]

[mouse]: https://camo.githubusercontent.com/09cbc14d7458e0e2c52f1a66fb8890152da978c9/68747470733a2f2f662e636c6f75642e6769746875622e636f6d2f6173736574732f3138322f35343631302f66326530656131382d356139612d313165322d383236642d6635663562663033663032652e676966

Aerijo commented:
Really? I always fill it out in the text, so I've never tried clicking it when viewing the preview.

Aerijo commented:
Besides, if they were going to be selective about which steps thay do, they'll probably still tick all the boxes regardless. I know I do that with some things (like the run in safe mode requirement; it probably won't change anything, unless you have a wacky setup).

yudai-nkt commented:
> it probably won't change anything, unless you have a wacky setup

This may be true, but issue reporters should follow the instruction as long as the issue template tells them to do so don't they?

Aerijo commented:
Yeah, they should. I guess my position is that one box is just as good, or better, than multiple boxes. This is how it is on the https://github.com/atom/atom repository.

Having said that, I will change it to multiple boxes if you prefer. It's not the biggest difference in the world, and I like to follow what others want for this package, while trying out my ideas on my own packages.

yudai-nkt commented:
Uh, I didn't mean to force multiple boxes and merged this only a few seconds after your reply. I just wanted to confirm the intention of the quoted text.

Thanks again for this PR (I hope we don't see MWEs posted as an image anymore lol)!

Aerijo commented:
@yudai-nkt While, I'm at it, I think I'll make a PR template as well. I had to tell the guy in #160 to update the changelog as well, so giving contributors a checklist would be helpful to everyone.

On a side note, as I say in the PR the proposed change closely resembles the existing patterns for citations, so I don't see any problem there. I'm just not sure how widely used the glossaries package is.
